### ISSUE TITLE
Disable ALL MR logging

### ISSUE THREAD

**Issue by westongpt** [magicalpanda/MagicalRecord#469]
Is there a way to disable all logging for MagicalRecord? I am trying to debug a non-MagicalRecord issue and its hard to parse with all of the  NSManagedObjectCalls.

I have #define MR_ENABLE_ACTIVE_RECORD_LOGGING 0

but it doesn't help.

**Comment by D-Wang** [comment]
@airdrummingfool  Hi, I set  MR_ENABLE_ACTIVE_RECORD_LOGGING=0 in the pod project to disable magical record logging. But when I set  want  to enable logging, I set MR_ENABLE_ACTIVE_RECORD_LOGGING=1 but it not work. How can I enable  it again?
